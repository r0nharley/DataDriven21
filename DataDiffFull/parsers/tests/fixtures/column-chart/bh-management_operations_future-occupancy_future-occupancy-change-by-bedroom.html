<svg version="1.1" class="highcharts-root" style="font-family:&quot;Lucida Grande&quot;, &quot;Lucida Sans Unicode&quot;, Arial, Helvetica, sans-serif;font-size:12px;" xmlns="http://www.w3.org/2000/svg" width="611" height="547">
    <desc>Created with Highstock 5.0.6</desc>
    <defs>
        <clipPath id="highcharts-zee5lcw-110">
            <rect x="0" y="0" width="535" height="463" fill="none"></rect>
        </clipPath>
        <clipPath id="highcharts-zee5lcw-111">
            <rect x="0" y="0" width="535" height="463" fill="none"></rect>
        </clipPath>
    </defs>
    <rect fill="#ffffff" class="highcharts-background" x="0" y="0" width="611" height="547" rx="0" ry="0"></rect>
    <rect fill="none" class="highcharts-plot-background" x="66" y="10" width="535" height="463"></rect>
    <g class="highcharts-grid highcharts-xaxis-grid ">
        <path fill="none" class="highcharts-grid-line" d="M 154.5 10 L 154.5 473" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 243.5 10 L 243.5 473" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 333.5 10 L 333.5 473" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 422.5 10 L 422.5 473" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 511.5 10 L 511.5 473" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 600.5 10 L 600.5 473" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 65.5 10 L 65.5 473" opacity="1"></path>
    </g>
    <g class="highcharts-grid highcharts-yaxis-grid ">
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 66 473.5 L 601 473.5" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 66 396.5 L 601 396.5" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 66 319.5 L 601 319.5" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 66 242.5 L 601 242.5" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 66 164.5 L 601 164.5" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 66 87.5 L 601 87.5" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 66 9.5 L 601 9.5" opacity="1"></path>
    </g>
    <rect fill="none" class="highcharts-plot-border" x="66" y="10" width="535" height="463"></rect>
    <g class="highcharts-axis highcharts-xaxis ">
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 154.5 473 L 154.5 483" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 243.5 473 L 243.5 483" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 333.5 473 L 333.5 483" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 422.5 473 L 422.5 483" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 511.5 473 L 511.5 483" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 601.5 473 L 601.5 483" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 65.5 473 L 65.5 483" opacity="1"></path>
        <path fill="none" class="highcharts-axis-line" stroke="#ccd6eb" stroke-width="1" d="M 66 473.5 L 601 473.5"></path>
    </g>
    <g class="highcharts-axis highcharts-yaxis ">
        <path fill="none" class="highcharts-axis-line" d="M 66 10 L 66 473"></path>
    </g>
    <g class="highcharts-series-group">
        <g class="highcharts-series highcharts-series-0 highcharts-column-series highcharts-color-undefined highcharts-tracker" transform="translate(66,10) scale(1 1)" width="463" height="535" clip-path="url(https://staging.rentlytics.com/bh-management/bi/operations-future-occupancy/visualization#highcharts-zee5lcw-111)">
            <rect x="19.5" y="-0.5" width="15" height="139" fill="#6EDA55" class="highcharts-point highcharts-negative" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="108.5" y="-0.5" width="15" height="117" fill="#6EDA55" class="highcharts-point highcharts-negative" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="197.5" y="-0.5" width="15" height="106" fill="#6EDA55" class="highcharts-point highcharts-negative" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="286.5" y="-0.5" width="15" height="115" fill="#6EDA55" class="highcharts-point highcharts-negative" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="375.5" y="-0.5" width="15" height="110" fill="#6EDA55" class="highcharts-point highcharts-negative" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="464.5" y="-0.5" width="15" height="437" fill="#6EDA55" class="highcharts-point highcharts-negative" stroke="#ffffff" stroke-width="1"></rect>
        </g>
        <g class="highcharts-markers highcharts-series-0 highcharts-column-series highcharts-color-undefined " transform="translate(66,10) scale(1 1)" width="463" height="535" clip-path="none"></g>
        <g class="highcharts-series highcharts-series-1 highcharts-column-series highcharts-color-undefined highcharts-tracker" transform="translate(66,10) scale(1 1)" width="463" height="535" clip-path="url(https://staging.rentlytics.com/bh-management/bi/operations-future-occupancy/visualization#highcharts-zee5lcw-111)">
            <rect x="36.5" y="-0.5" width="15" height="80" fill="#9b9bd7" class="highcharts-point highcharts-negative" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="126.5" y="-0.5" width="15" height="36" fill="#9b9bd7" class="highcharts-point highcharts-negative" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="215.5" y="-0.5" width="15" height="28" fill="#9b9bd7" class="highcharts-point highcharts-negative" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="304.5" y="-0.5" width="15" height="53" fill="#9b9bd7" class="highcharts-point highcharts-negative" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="393.5" y="-0.5" width="15" height="41" fill="#9b9bd7" class="highcharts-point highcharts-negative" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="482.5" y="-0.5" width="15" height="228" fill="#9b9bd7" class="highcharts-point highcharts-negative" stroke="#ffffff" stroke-width="1"></rect>
        </g>
        <g class="highcharts-markers highcharts-series-1 highcharts-column-series highcharts-color-undefined " transform="translate(66,10) scale(1 1)" width="463" height="535" clip-path="none"></g>
        <g class="highcharts-series highcharts-series-2 highcharts-column-series highcharts-color-undefined highcharts-tracker" transform="translate(66,10) scale(1 1)" width="463" height="535" clip-path="url(https://staging.rentlytics.com/bh-management/bi/operations-future-occupancy/visualization#highcharts-zee5lcw-111)">
            <rect x="54.5" y="-0.5" width="15" height="49" fill="#00cee6" class="highcharts-point highcharts-negative" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="143.5" y="-0.5" width="15" height="16" fill="#00cee6" class="highcharts-point highcharts-negative" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="233.5" y="-0.5" width="15" height="13" fill="#00cee6" class="highcharts-point highcharts-negative" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="322.5" y="-0.5" width="15" height="36" fill="#00cee6" class="highcharts-point highcharts-negative" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="411.5" y="-0.5" width="15" height="61" fill="#00cee6" class="highcharts-point highcharts-negative" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="500.5" y="-0.5" width="15" height="71" fill="#00cee6" class="highcharts-point highcharts-negative" stroke="#ffffff" stroke-width="1"></rect>
        </g>
        <g class="highcharts-markers highcharts-series-2 highcharts-column-series highcharts-color-undefined " transform="translate(66,10) scale(1 1)" width="463" height="535" clip-path="none"></g>
    </g>
    <g class="highcharts-legend" transform="translate(63,506)">
        <rect fill="none" class="highcharts-legend-box" rx="0" ry="0" x="0" y="0" width="485" height="26" visibility="visible"></rect>
        <g>
            <g>
                <g class="highcharts-legend-item highcharts-column-series highcharts-color-undefined highcharts-series-0" transform="translate(8,3)">
                    <text x="21" style="color:#333333;font-size:12px;font-weight:normal;cursor:pointer;fill:#333333;" text-anchor="start" y="15">
                        <tspan>60 Day Occ Change</tspan>
                    </text>
                    <rect x="2" y="4" width="12" height="12" fill="#6EDA55" rx="6" ry="6" class="highcharts-point"></rect>
                </g>
                <g class="highcharts-legend-item highcharts-column-series highcharts-color-undefined highcharts-series-1" transform="translate(171.03125,3)">
                    <text x="21" y="15" style="color:#333333;font-size:12px;font-weight:normal;cursor:pointer;fill:#333333;" text-anchor="start">
                        <tspan>30 Day Occ Change</tspan>
                    </text>
                    <rect x="2" y="4" width="12" height="12" fill="#9b9bd7" rx="6" ry="6" class="highcharts-point"></rect>
                </g>
                <g class="highcharts-legend-item highcharts-column-series highcharts-color-undefined highcharts-series-2" transform="translate(334.0625,3)">
                    <text x="21" y="15" style="color:#333333;font-size:12px;font-weight:normal;cursor:pointer;fill:#333333;" text-anchor="start">
                        <tspan>15 Day Occ Change</tspan>
                    </text>
                    <rect x="2" y="4" width="12" height="12" fill="#00cee6" rx="6" ry="6" class="highcharts-point"></rect>
                </g>
            </g>
        </g>
    </g>
    <g class="highcharts-axis-labels highcharts-xaxis-labels ">
        <text x="110.58333333333334" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:79px;text-overflow:clip;" text-anchor="middle" transform="translate(0,0)" y="492" opacity="1">
            <tspan>Studio</tspan>
        </text>
        <text x="199.75" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:79px;text-overflow:clip;" text-anchor="middle" transform="translate(0,0)" y="492" opacity="1">
            <tspan>1 BR</tspan>
        </text>
        <text x="288.9166666666667" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:79px;text-overflow:clip;" text-anchor="middle" transform="translate(0,0)" y="492" opacity="1">
            <tspan>2 BR</tspan>
        </text>
        <text x="378.08333333333337" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:79px;text-overflow:clip;" text-anchor="middle" transform="translate(0,0)" y="492" opacity="1">
            <tspan>3 BR</tspan>
        </text>
        <text x="467.25" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:79px;text-overflow:clip;" text-anchor="middle" transform="translate(0,0)" y="492" opacity="1">
            <tspan>4+ BR</tspan>
        </text>
        <text x="556.4166666666666" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:79px;text-overflow:clip;" text-anchor="middle" transform="translate(0,0)" y="492" opacity="1">
            <tspan>N\A</tspan>
        </text>
    </g>
    <g class="highcharts-axis-labels highcharts-yaxis-labels ">
        <text x="51" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:192px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="478" opacity="1">
            <tspan>-12.0%</tspan>
        </text>
        <text x="51" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:192px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="400" opacity="1">
            <tspan>-10.0%</tspan>
        </text>
        <text x="51" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:192px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="323" opacity="1">
            <tspan>-8.0%</tspan>
        </text>
        <text x="51" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:192px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="246" opacity="1">
            <tspan>-6.0%</tspan>
        </text>
        <text x="51" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:192px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="169" opacity="1">
            <tspan>-4.0%</tspan>
        </text>
        <text x="51" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:192px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="92" opacity="1">
            <tspan>-2.0%</tspan>
        </text>
        <text x="51" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:192px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="15" opacity="1">
            <tspan>0%</tspan>
        </text>
    </g>
</svg>