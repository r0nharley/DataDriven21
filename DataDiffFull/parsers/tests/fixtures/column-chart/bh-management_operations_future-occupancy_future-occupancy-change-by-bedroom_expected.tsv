Studio	60 Day Occ Change	-3.60259179266	#6EDA55
Studio	30 Day Occ Change	-2.07343412527	#9b9bd7
Studio	15 Day Occ Change	-1.26997840173	#00cee6
1 BR	60 Day Occ Change	-3.03239740821	#6EDA55
1 BR	30 Day Occ Change	-0.933045356371	#9b9bd7
1 BR	15 Day Occ Change	-0.414686825054	#00cee6
2 BR	60 Day Occ Change	-2.74730021598	#6EDA55
2 BR	30 Day Occ Change	-0.725701943844	#9b9bd7
2 BR	15 Day Occ Change	-0.336933045356	#00cee6
3 BR	60 Day Occ Change	-2.98056155508	#6EDA55
3 BR	30 Day Occ Change	-1.37365010799	#9b9bd7
3 BR	15 Day Occ Change	-0.933045356371	#00cee6
4+ BR	60 Day Occ Change	-2.85097192225	#6EDA55
4+ BR	30 Day Occ Change	-1.0626349892	#9b9bd7
4+ BR	15 Day Occ Change	-1.58099352052	#00cee6
N\A	60 Day Occ Change	-11.3261339093	#6EDA55
N\A	30 Day Occ Change	-5.90928725702	#9b9bd7
N\A	15 Day Occ Change	-1.84017278618	#00cee6